sw/device/silicon_creator/lib/base/in_memory_epmp.h

```diff
@@ -0,0 +1,24 @@
+// Copyright lowRISC contributors.
```

Comment:
nit: can we call the header source pair `static_critical_epmp_state.c` and `static_critical_epmp_state.h` for the sake of consistency with other definitions? You also need to fix the include guard and `bazel run //:buildifier_fix`.

Reply:
Fixed